### ISSUE TITLE
CCEDICT error at V.2.0.0

### ISSUE THREAD

**Issue by Dans-io** [nieldlr/Hanzi#38]
Hello @nieldlr,

We talked earlier today for the 2.0 NPM Launch. Thank you for that.
I was trying it just now and found two things. First, on the web, I tried to do

`hanzi.decompose('爱');`

which resulted in

```
Uncaught TypeError: Cannot read property 'indexOf' of undefined
    at Object.start (hanzidecomposer.js:21)
    at Object.start (module.js:5)
    at App.componentDidMount (App.js:28)
    at ReactCompositeComponent.js:262
    at measureLifeCyclePerf (ReactCompositeComponent.js:73)
    at ReactCompositeComponent.js:261
    at CallbackQueue.notifyAll (CallbackQueue.js:74)
    at ReactReconcileTransaction.close (ReactReconcileTransaction.js:78)
    at ReactReconcileTransaction.closeAll (Transaction.js:207)
    at ReactReconcileTransaction.perform (Transaction.js:154)
```

Later I tried it in React Native, where I got

`Unable to resolve module './cedict_ts.u8' from '.../node_modules/hanzi/lib/data/index.js': could not resolve '.../node_modules/hanzi/lib/data/cedict_ts.u8' as a file nor as a folder`

For the React Native issue, I checked the code but couldn't find anything wrong. I also researched for .u8 files in React Native to see if they could cause a problem and again couldn't find anything.

I'll keep looking for an answer but I wanted to post it here in case you or @zurawiki can help me out with this.

Thanks a lot,
Dan.

**Comment by zurawiki** [comment]
Hi Dan,

I'm don't know the react-native build pipeline too well but I have a hunch that the data files with a non-`.js` extension are not being picked up by `webpack`. 

I pushed a branch with a fix on my fork. Can you try using this branch here and see if the issue goes away? If it does, then I'll send out a PR for the fix.

See my branch at: https://github.com/zurawiki/Hanzi/tree/rmz-data-rn

The following command should install my branch as the npm package
```sh
npm install github:zurawiki/Hanzi#rmz-data-rn
```

**Comment by Dans-io** [comment]
Oh wow that actually did it. Everything works perfectly in React Native. Thank you so much!

**Comment by nieldlr** [comment]
So awesome @zurawiki. Thanks for this.

This should now be merged (#40) & published under v2.1.0 (#41) @Dans-io 👍

Let me know if there's anything else.
